net-print/kyocera-1x2x-mfp-driver/files/rastertokpsl-fix.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# This file is a slightly modified version of 'rastertokpsl_wrapper' script
+# by Maxim Norin. See https://github.com/mnorin/kyocera-ppd-installer for
```

Comment:
Actually, the upstream repo indicates that the wrapper has a different author. But I guess it's acceptable as long as there's URI. I've filed mnorin/kyocera-ppd-installer#2 to get clear licenses statement from upstream.

Reply:
Please check https://github.com/mnorin/kyocera-ppd-installer for licenses information and let me know if everything is clear enough

Reply:
@mnorin 

Thank you, the license is clear and seems to compatible with portage. However, if the code originally had a different author, we should mention him as well (an ask if he is fine with GPLv2). This makes things complicated :(.


@mnorin

Reply:
The only information I have about wrapper author is the link to Yandex.Market comment https://market.yandex.ru/product/8523545/forum?hid=138608&comment=child-7-1477811